test/test_genbank_sbol3_direct.py

```diff
@@ -209,6 +209,24 @@ def test_round_trip_iGEM_BBF10K(self):
         )
         self._test_round_trip_genbank(genbank_file)
     
+    def test_round_trip_structured_comments(self):
+        """Test ability to correctly round trip genbank test files in the iGEM distribution which have 
+        comment and structured comment annotations.
+        """
+        genbank_file = (
+            Path(__file__).parent / "test_files" / "sbol3_genbank_conversion" / "test_structured_comments.gb"
```

Comment:
In a future world, perhaps by someone else, we could create a variable to hold the `Path` to the `sbol3_genbank_conversion` directory. In general, watch for these patterns and extract them to common code so they are easier to update in the future.

```python
TEST_DIR = Path(__file__).parent / "test_files" / "sbol3_genbank_conversion"
```

Note: _This is not a requested change, just a remark on the code._

Reply:
Right thanks, this makes sense